uploader/src/main/resources/lambda-template.yaml

```diff
@@ -19,6 +19,9 @@
         CONFIG_KEY: !Sub
           - "${Stage}/media-atom-maker.private.conf"
           - { Stage: !Ref Stage }
+        CREDENTIALS_KEY: !Sub
+          - "${Stage}/youtube-service-account.json"
+          - { Stage: !Ref Stage }
```

Comment:
Does the lambda have read permission for this file?

Reply:
Yes, thanks to https://github.com/guardian/media-atom-maker/blob/2c67a5964df81baf1bc0c68314de2caa7ad8f721/uploader/src/main/resources/cfn-template.yaml#L78